package.json

```diff
@@ -31,6 +31,7 @@
   "devDependencies": {
     "@babel/cli": "^7.8.3",
     "@babel/core": "^7.8.3",
+    "@babel/plugin-proposal-private-methods": "^7.8.3",
```

Comment:
Does it make sense to review these dependencies and why some key packages are included @wswebcreation? Perhaps the @wdio dependencies in particular?

Reply:
Hmm, no I don't think this it is important to review the dependencies, the testcode is enough 😉